contracts/HashRegistrarSimplified.sol

```diff
@@ -130,8 +130,7 @@ contract Registrar {
     //   Auction -> Reveal
     //   Reveal -> Owned
     //   Reveal -> Open (if nobody bid)
-    //   Owned -> Forbidden (invalidateName)
-    //   Owned -> Open (releaseDeed)
+    //   Owned -> Open (releaseDeed or invalidateName)
```

Comment:
Does it make sense to also change line 106: `enum Mode { Open, Auction, Owned, Forbidden, Reveal, NotYetAvailable }`?

Pro: removes redundant code
Con: might break any code that expected Reveal and NotYetAvailable to be numbers 4 and 5